aspnetcore/security/enforcing-ssl.md

```diff
@@ -127,7 +150,7 @@ The following code:
 
```

Comment:
"`UseHsts` isn't recommended in development because the HSTS header" -> because the HSTS settings are highly cacheable..." It's not the header being cached, it's the values.